install/error-handling.sh

```diff
@@ -188,6 +190,10 @@ cleanup() {
             --arg lineno "$lineno" \
             '{"filename": $filename, "function": $function, "lineno": $lineno|tonumber}'
         )
+        # If we're in the stacktrace of the file we failed on, we can add a context line with the command run that failed
+        if [[ $i -eq 1 ]]; then
+          JSON="{\"context_line\": \"$cmd\", ${JSON:1}"
```

Comment:
I'd rather this be done with jq as well since jq will handle any escaping that is needed if `$cmd` has characters that would be a problem.